lib/secretsService.js

```diff
@@ -0,0 +1,49 @@
+const apiUrl = 'https://optic-web.herokuapp.com/api'
```

Comment:
we may want to make this an env var variable which is embedded in the compiled app instead of hardcoding it

Reply:
Sure, this is the stuff I pulled from the previous PR, will refactor in the next one.